### DISCUSSION TITLE
20240912 Weekly Development Meeting

### DISCUSSION THREAD

**Discussion by josecelano** [nautilus-cyberneering/.github#107] [Meetings]
- Time:  9:30 in Canary Islands (WET (UTC+0)).
- Time: 10:30 in Vaduz (CET (UTC+1)).

Location:
<https://meeting.blockfinance-eco.li/NautilusCyberneeringWeeklyDevMeeting>

Expected duration: 2 to 4 hours with breaks

Communicated attendants:

- Cameron Garnham(@da2ce7)  - x 
- Jose Celano(@josecelano) - x
- Constantin Bosse(@cgbosse) - x
- Mario Melchor(@mario-nt) - x
- Graeme Byrne(@grmbyrn) - x

-------------------------------------------------------------

## Agenda:

1. Everyone will take their turn and:
- Go over his notes added as comments for the meeting which will have been added in advance to give an update of all the work done during the previous week indicating.
	- The issues worked on
	- The progress made on them
	- Other things done
- Inform of any faced challenges and request help if necessary to work on specific issues.
- Highlight the encountered difficulties, especially if there is a risk of them causing delays or affecting end product's quality.
- Communicate what will be the next week's tasks.
- List any new topics for further discussion in an additional Technical meeting that he noted for future discussion in comments to this discussion either before the meeting or after his turn so that they get considered.

During the meeting every team member will take their turns and follow this sequence.
The topics for further discussion will be noted in advance by each the team member and to the meetings agenda.

2. At the end of the meeting if there any new technical topics to be discussed further it will be decided to
set another meeting date for the discussion of these or to hold the meeting hereafter.

Also the persons to be part of the meeting will be decided on so that the others will be free to continue their own work.

**Comment by josecelano** [comment]
### Torrust Website

- New blog post: [Hash2Torrent - Retrieve Torrent Files Effortlessly!](https://torrust.com/hash2torrent-retrieve-torrent-files-effortlessly)

### Torrust Tracker

- Review [Tracker Checker issues](https://github.com/torrust/torrust-tracker/issues/669) and close them (most of them have already been implemented by @da2ce7).
- Issues: 
   - https://github.com/torrust/torrust-tracker/issues/569
   - https://github.com/torrust/torrust-tracker/issues/675 **WIP**
- Research: Bencode to JSON conversion (needed for some issues related to the Tracker Checker). See https://github.com/torrust/teps/discussions/15#discussioncomment-10611909

### Torrust Demo:

Opened new issues after checking logs and tracker DB: 

- https://github.com/torrust/torrust-demo/issues/12
- https://github.com/torrust/torrust-demo/issues/14

### Torrust Index

- Fix bug: https://github.com/torrust/torrust-index/issues/724
- Meeting with Mario to solve a problem with E2E tests. E2E test execution is too complex now. There are four groups of tests:
  - SQLite
    - Tracker in public mode
    - Tracker in private mode
  - MySQL
    - Tracker in public mode
    - Tracker in private mode
- https://github.com/torrust/torrust-index/issues/728  

### Next

- Review texts for the new Torrust site.
- Continue closing minor (easy) issues in the Tracker.

**Comment by mario-nt** [comment]
### Index
	
- [https://github.com/torrust/torrust-index/issues/615 - tests](https://github.com/torrust/torrust-index/pull/718/)

### Other

- Review pending emails and Github notifications
- 1 to 1 with Jose about E2E tests (Using Docker, discussing errors and configs, troubleshooting errors)
